src/python/TaskWorker/Actions/DagmanResubmitter.py

```diff
@@ -85,71 +83,43 @@ def executeInternal(self, *args, **kwargs): #pylint: disable=unused-argument
                   'RequestCpus'        : 'numcores',
                   'JobPrio'            : 'priority'
                  }
-        overwrite = False
         for taskparam in params.values():
-            if ('resubmit_'+taskparam in task) and task['resubmit_'+taskparam] != None:
-                # py3: we can revert changes coming from #5317
+            if ('resubmit_'+taskparam in task) and task['resubmit_'+taskparam] is not None:
+                # the following seems to Stefano a very complicated way to handle
+                # in different (proper) way the cases where resubmit_xxxx is list and
+                # where it is a number. Should be verified and simplified
                 if isinstance(task['resubmit_'+taskparam], list):
                     ad[taskparam] = task['resubmit_'+taskparam]
-                if taskparam != 'jobids':
-                    overwrite = True
-
         if ('resubmit_jobids' in task) and task['resubmit_jobids']:
-            with HTCondorUtils.AuthenticatedSubprocess(proxy, tokenDir,
-                                                       logger=self.logger) as (parent, rpipe):
-                if not parent:
-                    schedd.edit(rootConst, "HoldKillSig", 'SIGKILL')
-                    ## Overwrite parameters in the os.environ[_CONDOR_JOB_AD] file. This will affect
-                    ## all the jobs, not only the ones we want to resubmit. That's why the pre-job
-                    ## is saving the values of the parameters for each job retry in text files (the
-                    ## files are in the directory resubmit_info in the schedd).
-                    for adparam, taskparam in params.items():
-                        if taskparam in ad:
-                            schedd.edit(rootConst, adparam, ad.lookup(taskparam))
-                        elif task['resubmit_'+taskparam] != None:
-                            schedd.edit(rootConst, adparam, str(task['resubmit_'+taskparam]))
-                    schedd.act(htcondor.JobAction.Hold, rootConst)
-                    schedd.edit(rootConst, "HoldKillSig", 'SIGUSR1')
-                    schedd.act(htcondor.JobAction.Release, rootConst)
-        elif overwrite:
```

Comment:
Could you explain a bit more why we need this in the past?

I understand that it is not used anymore; at least I did not see any logs from this section on prod machine.

Reply:
took me a long time to decide that we do not need it !  And I did look at code history boing back many years.
I think it is unused since a very long time. Apparently there was a time when `crab resubmit` would result in calling `DagmanResubmitter` without a list of jobids to be resubmitted. Then people added the possibility to resubmit just some jobs and at some point code became what it is now, that DagmanResubmitter always starts with a list of JobIds.
I can see use cases for "only resubmit this/these jobs", and I guess people decided that making the list of jobs in the client every time was simpler. Unfortunately they lefts old code around in the TW. 

We may want to go back to allow a 'resubmit w/o a list' when rewrite resubmission as per #6270 But that will start with defining a new semantic for resubmissions !!

Reply:
Wow. Thanks!